lib/rubocop/cop/rails/rake_environment.rb

```diff
@@ -41,6 +41,18 @@ def on_block(node)
           end
         end
 
+        def autocorrect(node)
+          lambda do |corrector|
+            task_name = node.arguments[0]
+            corrected_task_name =
+              task_name.source.delete(':').delete("'").delete('"')
```

Comment:
Probably the code can be made compact.

```suggestion
            task_name = node.first_argument
            corrected_task_name = task_name.source.delete(':|\'|"')
```